tools/chpldef/Transport.cpp

```diff
@@ -97,7 +97,7 @@ bool Transport::sendJsonBlocking(Server* ctx, std::ostream& os,
                                  const JsonValue& json) {
   const bool pretty = false;
   std::string s = jsonToString(json, pretty);
-  os << std::move(s);
+  os << "Content-Length: " << s.size() << "\r\n\r\n" << std::move(s);
```

Comment:
Something makes me think that moving the string isn't needed here. Most likely the `operator <<` only accepts the string by const ref. Rvalue refs can be made into const refs. But that's exactly what would happen if you just passed the string normally.